src/items/class-features/dreamer.json

```diff
@@ -0,0 +1,120 @@
e danger, though it can’t communicate the nature of the threat to you.</p><h2>@UUID[Compendium.sfrpg.class-features.4Sfvo9sMJKXHItQL]{Dream Manifestation (Su) - 6th Level}</h2><p>You can spend 1 Resolve Point as a full action to pull a concept or image from dreamspace and manifest it in the physical realm. This ability reproduces the effects of a spell of up to the highest level you can cast and can come from any spell list. You control the appearance of the dream manifestation.</p><h2>@UUID[Compendium.sfrpg.class-features.O7jG2sX6am0Pl1S9]{Fortified Imagination (Su) - 9th Level}</h2><p>You’ve experienced countless nightmares, preparing you to handle anything. You gain a +2 insight bonus to @Check[type:will] saves against mind-affecting spells and abilities. As a swift action, you can spend 1 Resolve Point to grant this bonus to an ally who can perceive you.</p><h2>@UUID[Compendium.sfrpg.class-features.y6r1wbn3T0hAG4e0]{Dream Sending (Su) - 12th Level}</h2><p>Once per day while inside dreamspace, you can briefly communicate with creatures at great distances. This creature can be anywhere in the universe, including on another plane, as long as you’re familiar with them. You can speak freely with any ally linked by your telepathic bond class feature when creating a dream sending. Creatures that aren’t linked to you by a @UUID[Compendium.sfrpg.class-features.ox6Am16XZqBXJNT9]{Telepathic Bond (Sp)} can attempt a @Check[type:will] save with a DC equal to [[15+floor( @classes.mystic.levels / 2 )]]{15 + one-half your class level} to resist the dream sending. A creature can choose to fail this save. On a failure, the creature experiences a dream containing a brief message of 25 words or less the next time it rests. The message can be verbal or nonverbal, and you control the aesthetic, imagery, and events of this dream. The creature wakes remembering your message, though you can’t influence a creature’s actions or attitude toward you using this ability.</p><h2>@UUID[Compendium.sfrpg.class-features.FkLyMHbGPGwIsPJA]{Dream Blessing (Su) - 15th Level}</h2><p>Once per day, you can send a dream blessing to a creature with which you’re familiar. You control the aesthetic, imagery, and events of this dream. The creature experiences the dream when they next rest. For the next 24 hours, the creature benefits from a +2 bonus to a saving throw or skill check of your choice (this choice must be related to the dream’s subject matter) and can reroll a failed check of that type once per day</p><h2>@UUID[Compendium.sfrpg.class-features.HqBLb5H1BN81uHuB]{Visitation (Su) - 18th Level}</h2><p>Once per day, you can spend 1 Resolve Point as a full action to physically travel to the location of a specific dreamer through dreamspace. The target dreamer can be anywhere in the universe, including on another plane, as long as you’re familiar with them. The visitation lasts for a number of minutes equal to [[floor( @classes.mystic.levels / 2 )]]{one-half your character level}.</p>"
```

Comment:
Still a bunch of inline DCs here

Reply:
I think as long as the will save buttons have been removed and the formulas are spelled out in the text, it's okay to leave the automated DC calculations.